app/classifier/tasks/drawing/grid-editor.jsx

```diff
@@ -0,0 +1,168 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import AutoSave from '../../../components/auto-save';
+
+class GridEditor extends React.Component {
+  static propTypes = {
```

Comment:
Same comment about the static declaration here. This will have to be instead:
```
GridEditor.propTypes = {};
```

Reply:
You can ignore this, but the ESLint rule set needs updating to reflect that the babel transforms are supporting this now.